services/observability/CHANGELOG.md

```diff
@@ -0,0 +1,123 @@
+## v0.12.0 (2024-08-21)
+- **Breaking change**: STACKIT Argus service has been renamed to STACKIT Observability
```

Comment:
I would start versioning from `v0.1.0` as this is a new service (similar to https://github.com/stackitcloud/stackit-sdk-go/blob/main/services/authorization/CHANGELOG.md)

Reply:
We could e.g. link the `argus` changelogs in case the user wants to look at them

Reply:
Changed version and linked the old argus changelogs like the authorization example